packages/gatsby-plugin-typescript/src/gatsby-node.js

```diff
@@ -1,49 +1,58 @@
+const babelPluginRemoveQueries = require(`babel-plugin-remove-graphql-queries`)
 const { transpileModule } = require(`typescript`)
-const path = require(`path`)
+const resolve = require(`./resolve`)
 
 const test = /\.tsx?$/
 const compilerDefaults = {
   target: `esnext`,
   experimentalDecorators: true,
   jsx: `react`,
+  module: `es6`,
 }
 
 module.exports.resolvableExtensions = () => [`.ts`, `.tsx`]
 
-module.exports.modifyWebpackConfig = ({ config }, { compilerOptions }) => {
-  // CommonJS to keep Webpack happy.
-  const copts = Object.assign({}, compilerDefaults, compilerOptions, {
-    module: `commonjs`,
-  })
-
-  // React-land is rather undertyped; nontrivial TS projects will most likely
-  // error (i.e., not build) at something or other.
-  const opts = { compilerOptions: copts, transpileOnly: true }
+module.exports.modifyWebpackConfig = (
+  { boundActionCreators, loaders },
+  { compilerOptions, ...options }
+) => {
+  // gatsby removes graphql queries from source code since they are processed
```

Comment:
Just capitalize Gatsby :-) Just trying to add some annoying maintainer comments haha :-P

I want Prettier for comments basically.